### ISSUE TITLE
Unable to find module @mux/mux-node/_shims/auto/jwt

### ISSUE THREAD

**Issue by joselevelsup** [muxinc/mux-node-sdk#355]
Hey all! Appreciate all the hard work on this SDK. 
I keep getting an issue whenever I run my server with the Mux SDK. I am using Bun 1.0.33 and mux sdk 8.2.2. This is the error I keep getting: 

`error: Cannot find module "@mux/mux-node/_shims/auto/jwt" from "<my-project-directory>/node
_modules/@mux/mux-node/resources/jwt.mjs"`

If someone knows how I should go about fixing this, that would be great. Thanks!

**Comment by rattrayalex** [comment]
Hmm. Does it help to add this line of code before your first import of the library?

```ts
import '@mux/mux-node/shims/web'
```

**Comment by joselevelsup** [comment]
Nope. I get the same error still.

**Comment by rattrayalex** [comment]
Gotcha. Okay, we'll look into it this coming week!

**Comment by rattrayalex** [comment]
Can you share how you're importing/using the library?

**Comment by joselevelsup** [comment]
Sure no problem! 

```javascript
import '@mux/mux-node/shims/web';
import Mux from "@mux/mux-node";
import * as env from "env-var";

const mux = new Mux({
  tokenId: env.get("MUX_TOKEN").required().asString(),
  tokenSecret: env.get("MUX_SECRET").required().asString()
});

export default mux;
```

It's nothing crazy. After doing this, I just import it into one of my files to get a list of live streams (nothing else) and that's when I ran into this error.

**Comment by RobertCraigie** [comment]
This has been fixed in `v8.3.1`!
